lib/ice_cube/time_util.rb

```diff
@@ -1,5 +1,5 @@
-require "date"
-require "time"
+require 'date'
+require 'time'
```

Comment:
Changing the type of quotes seems unnecessary. Double quotes are consistent with the existing style.